io.github.ctlcltd.e2se.yaml

```diff
@@ -0,0 +1,43 @@
+app-id: io.github.ctlcltd.e2se
+runtime: org.kde.Platform
+runtime-version: '6.4'
+sdk: org.kde.Sdk
+command: e2-sat-editor
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=x11
+  - --socket=cups
```

Comment:
do you really need this?

Reply:
this software manages channel lists, uses FTP and reads/writes channel lists, it can also print them